src/test/javascript/spec/component/modeling-assessment/modeling-assessment.component.spec.ts

```diff
@@ -165,6 +181,60 @@ describe('ModelingAssessmentComponent', () => {
         expect(comp.feedbacks).toEqual(mockFeedbacks);
     });
 
+    it('should filter references by result feedbacks', () => {
+        expect(comp.referencedFeedbacks).toBeEmpty();
+        expect(comp.feedbacks).toBe(undefined);
+
+        comp.umlModel = mockModel;
+        comp.resultFeedbacks = mockFeedbacks;
+
+        expect(comp.referencedFeedbacks).toEqual([mockFeedbackWithReference, mockFeedbackInvalid]);
+        expect(comp.feedbacks).toBe(mockFeedbacks);
+    });
+
+    it('should calculate drop info', () => {
+        const spy = jest.spyOn(translatePipe, 'transform');
+        comp.umlModel = mockModel;
+        comp.resultFeedbacks = [mockFeedbackWithGradingInstruction];
+
+        expect(spy).toHaveBeenCalledWith('artemisApp.assessment.messages.removeAssessmentInstructionLink');
```

Comment:
Could we add an assertion on how many times the spy was called as well?
```js
expect(spy).toHaveBeenCalledTimes(...);
```

Reply:
Added the `toHaveBeenCalledTimes` assertion.